lib/factory_bot_profiler/collector.rb

```diff
@@ -30,6 +30,10 @@ def highest_average_time(n = 1)
       take_factory_values_by(:average_time, n)
     end
 
+    def all_factories
```

Comment:
I added an `each` over in https://github.com/composerinteralia/factory_bot_profiler/compare/merging?expand=1 that you might also be able to use. This class is starting to feel slightly enumarable-ish. That's perhaps a clue to a better name (Collector doesn't seem quite right to me for some reason).